crates/oxc_minifier/src/peephole/remove_unused_expression.rs

```diff
@@ -626,6 +650,31 @@ impl<'a> PeepholeOptimizations {
         false
     }
 
+    fn fold_expression_in_mathematical_operation(
+        &self,
+        e: &mut Expression<'a>,
+        state: &mut State,
+        ctx: Ctx<'a, '_>,
+    ) -> bool {
+        match e {
+            Expression::UnaryExpression(unary_expr) => {
+                if unary_expr.operator.is_arithmetic() {
+                    *e = unary_expr.argument.take_in(ctx.ast.allocator);
+                    state.changed = true;
+                }
+            }
```

Comment:
I think the unary `+` cannot be removed for comparisons.
```js
+s > 0 // error when s is bigint
s > 0 // no error
```

Reply:
There's a related implementation here: 
https://github.com/oxc-project/oxc/blob/691c2ab0740e2b232dbaced47feb66b2dbb73be8/crates/oxc_minifier/src/peephole/substitute_alternate_syntax.rs#L304-L333
Maybe it can be improved.